tmk_core/protocol/chibios/usb_main.c

```diff
@@ -534,40 +527,33 @@ static void usb_event_cb(USBDriver *usbp, usbevent_t event) {
                 }
                 qmkusbConfigureHookI(&drivers.array[i].driver);
             }
-            osalSysUnlockFromISR();
-            if (last_suspend_state) {
+            if (was_suspended) {
                 usb_event_queue_enqueue(USB_EVENT_WAKEUP);
+                was_suspended = false;
             }
             usb_event_queue_enqueue(USB_EVENT_CONFIGURED);
-            return;
+            break;
         case USB_EVENT_SUSPEND:
-            /* Falls into.*/
+            was_suspended = true;
         case USB_EVENT_UNCONFIGURED:
-            /* Falls into.*/
```

Comment:
maybe keep these comments for readability?

Reply:
Hmmn, I'm not convinced that it actually adds readability. Though that is ofc subjective.